docs/core.md

```diff
@@ -5,7 +5,7 @@ For async behavior, refer [./async.md](async.md).
 
 ## atom
 
-`atom` is a function to create an atom config. It's an object and the object identity is important. You can create it from everywhere. Once created, you shouldn't modify the object. (Note: There might be an advanced use case to mutate atom configs after creation. At the moment, it's not officially supported though.)
+`atom` is a function to create an atom config. It's an object that can be created anywhere however, once created, the object cannot be modified (immutable). (Note: There might be an advanced use case to mutate atom configs after creation. At the moment, it's not officially supported.)
```

Comment:
Please revert it or suggest a different one. I admit the original version is not very good, but the modified version doesn't seem correct. The atom object is mutable and can be modified, but it shouldn't, or you'd expect an unexpected behavior.

Reply:
Reverted content to original with very minor word modifications (same with other content)